File: frontend/src/@types/models/phone_subscriber/PhoneSubscriber.ts
export default interface PhoneSubscriber {
    erp_contract_id: number;
    name: string;
    phone_number: string;
    plan_name: string | null;
    product_name: string | null;
    summit_license: string | null;
    summit_category: string | null;
    summit_partition: string | null;
    city: string;
}

File: snmpagent/app/models/PhoneSubscriber.py
from django.db import models

class PhoneSubscriber(models.Model):
   erp_contract_id = models.IntegerField(blank=True, null=True)

   name = models.CharField(max_length = 100)
   phone_number = models.CharField(max_length = 50)
   plan_name = models.CharField(max_length = 150, null=True)
   product_name = models.CharField(max_length = 150, null=True)
   summit_license = models.CharField(max_length = 10, null=True)
   summit_category = models.CharField(max_length = 200, null=True)
   summit_partition = models.CharField(max_length = 200, null=True)
   city = models.CharField(max_length = 50, null=False)
   
   class Meta:
      db_table = "phone_subscriber"